CMakeLists.txt

```diff
@@ -116,14 +116,10 @@ else()
   endif()
 
   # warning level for edge specific source files 
-  set (EDGE_WARNINGS -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wno-deprecated-declarations)
+  set (EDGE_WARNINGS -Wextra)

```

Comment:
EDGE_WARNINGS is a bit vague, can this be renamed to EGDE_WARNING_LEVEL?

Reply:
Agreed, updated the various CMake files